src/libraries/Microsoft.Bcl.AsyncInterfaces/Directory.Build.props

```diff
@@ -1,7 +1,13 @@
-<Project>
+﻿<Project>
   <Import Project="..\Directory.Build.props" />
   <PropertyGroup>
     <StrongNameKeyId>Open</StrongNameKeyId>
     <!-- This assembly should never be placed inbox as it is only for downlevel compatibility. -->
+    <PackageDescription>Provides the IAsyncEnumerable&lt;T&gt; and IAsyncDisposable interfaces and helper types for .NET Standard 2.0. This package is not required starting with .NET Standard 2.1 and .NET Core 3.0.</PackageDescription>
   </PropertyGroup>
+  <ItemGroup>
+    <PackageDescriptionCommonType Include="System.IAsyncDisposable" />
```

Comment:
Couldn't we have a way to make `PackageDescription` and common types be part of the same item/property declaration?

Reply:
I considered inlining the types into the description. At the time when I looked at it (yesterday) I was against it as the "Commonly Used Types:" header string would be duplicates across all properties. Giving this a second thought, it might be easier to just do that...

Reply:
Sounds good, thanks for elaborating.

Reply:
Just updated the PR. Fortunately I didn't do this manually before :D